parse.go

```diff
@@ -119,9 +120,13 @@ func NewParser(in io.Reader, bytesToRead int64, frameChannel chan *frame.Frame)
 		frameChannel: frameChannel,
 	}
 
-	elems, err := p.readHeader()
-	if err != nil {
-		return nil, err
+	elems := []*Element{}
+
+	if !optSet.skipHeaderRead {
```

Comment:
In the case that the header is skipped, the parser will default to using Little Endian Implicit since no transfer syntax tag will be found in the empty dataset on L141 below. Is this tolerable for your use case? 

In the docstring for the SkipHeaderRead, we should mention this default behavior as well imo.

Reply:
This works fine for my use case. I've updated the docstring to reflect this behaviour.